packages/test/test-end-to-end-tests/src/test/codePropsal.spec.ts

```diff
@@ -219,12 +198,8 @@ describeNoCompat("CodeProposal.EndToEnd", (getTestObjectProvider) => {
                 maps.push(map);
             }
         }
-        const waiters: Promise<void>[] = [];
-        for (const map of maps) {
-            waiters.push(...keys.map(async (k) => mapWait(map, k)));
-        }
 
-        await Promise.all([provider.ensureSynchronized(), ...waiters]);
+        await Promise.all([provider.ensureSynchronized()]);
```

Comment:
```suggestion
        await provider.ensureSynchronized();
```

I think removing the waits here is OK based on my read of it, but the test is a little strange.  These waits imply that the test author didn't trust `provider.ensureSynchronized()` to actually ensure synchronization -- maybe dig a bit to see why that was?

Reply:
i don't think it existed. this change looks fine

Reply:
@anthony-murphy  I checked the function `ensureSynchronized()` has already returned a Promise object. Do we still need an extra `Promise.all()` out of it? 
![image](https://user-images.githubusercontent.com/86381878/178800986-e03cf45c-e020-4c7d-ab83-7954cdccf1b9.png)

Reply:
You don't need the Promise.all when there's a single promise, it can be removed.